backend/de.metas.adempiere.adempiere/migration/src/main/sql/postgresql/system/10-de.metas.adempiere/5637900_sys_gh11302_process_remove_old_hu.sql

```diff
@@ -0,0 +1,131 @@
+;
+
+-- 2022-05-05T07:38:13.545Z
+-- I forgot to set the DICTIONARY_ID_COMMENTS System Configurator
+INSERT INTO AD_Scheduler (AD_Client_ID,AD_Org_ID,AD_Process_ID,AD_Role_ID,AD_Scheduler_ID,Created,CreatedBy,EntityType,Frequency,FrequencyType,IsActive,IsIgnoreProcessingTime,KeepLogDays,ManageScheduler,Name,Processing,SchedulerProcessType,ScheduleType,Status,Supervisor_ID,Updated,UpdatedBy) VALUES (0,0,585049,0,550087,TO_TIMESTAMP('2022-05-05 09:38:13','YYYY-MM-DD HH24:MI:SS'),100,'D',1,'D','Y','N',7,'N','remove_hu_entries_between_dates','N','P','F','NEW',100,TO_TIMESTAMP('2022-05-05 09:38:13','YYYY-MM-DD HH24:MI:SS'),100)
+;
+
+-- 2022-05-05T07:38:33.308Z
+-- I forgot to set the DICTIONARY_ID_COMMENTS System Configurator
+INSERT INTO AD_Scheduler_Para (AD_Client_ID,AD_Org_ID,AD_Process_Para_ID,AD_Scheduler_ID,AD_Scheduler_Para_ID,Created,CreatedBy,IsActive,ParameterDefault,Updated,UpdatedBy) VALUES (0,0,542257,550087,540038,TO_TIMESTAMP('2022-05-05 09:38:33','YYYY-MM-DD HH24:MI:SS'),100,'Y','100',TO_TIMESTAMP('2022-05-05 09:38:33','YYYY-MM-DD HH24:MI:SS'),100)
+;
+
+-- 2022-05-05T07:38:56.861Z
+-- I forgot to set the DICTIONARY_ID_COMMENTS System Configurator
+INSERT INTO AD_Scheduler_Para (AD_Client_ID,AD_Org_ID,AD_Process_Para_ID,AD_Scheduler_ID,AD_Scheduler_Para_ID,Created,CreatedBy,IsActive,ParameterDefault,Updated,UpdatedBy) VALUES (0,0,542256,550087,540039,TO_TIMESTAMP('2022-05-05 09:38:56','YYYY-MM-DD HH24:MI:SS'),100,'Y','24',TO_TIMESTAMP('2022-05-05 09:38:56','YYYY-MM-DD HH24:MI:SS'),100)
+;
+
+-- 2022-05-05T08:07:28.389Z
+-- I forgot to set the DICTIONARY_ID_COMMENTS System Configurator
+UPDATE AD_Process SET Description='Removes all handling unit entries that have an inactive status and are older than the specified months. The placeholder is used in other tables linking to the removed handling unit.',Updated=TO_TIMESTAMP('2022-05-05 10:07:28','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_Process_ID=585049
+;
+
+-- 2022-05-05T08:07:49.128Z
+-- I forgot to set the DICTIONARY_ID_COMMENTS System Configurator
+UPDATE AD_Process_Para SET DefaultValue='100', IsMandatory='Y',Updated=TO_TIMESTAMP('2022-05-05 10:07:49','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_Process_Para_ID=542257
+;
+
+
+-- 2022-05-05T12:12:35.109Z
+-- I forgot to set the DICTIONARY_ID_COMMENTS System Configurator
+UPDATE AD_Scheduler SET CronPattern=NULL, Description='Removes old, inactive handling units', Name='Remove old Handling Units',Updated=TO_TIMESTAMP('2022-05-05 14:12:35','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_Scheduler_ID=550087
+;
+
+-- 2022-05-05T12:12:48.784Z
+-- I forgot to set the DICTIONARY_ID_COMMENTS System Configurator
+UPDATE AD_Scheduler SET CronPattern=NULL, IsActive='N',Updated=TO_TIMESTAMP('2022-05-05 14:12:48','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_Scheduler_ID=550087
```

Comment:
please use the "cron" FrequencyType and set a cron-pattern such that it would be run (if it was active) some time iduring the night.
reason: right now you would ahve 1 day between a process end and the next start. so the start times would move and eventuell the process would run at noon